packages/virtualized-lists/Lists/VirtualizedList.js

```diff
@@ -1969,7 +1970,9 @@ class VirtualizedList extends StateSafePureComponent<Props, State> {
 
 const styles = StyleSheet.create({
   verticallyInverted: {
-    transform: [{scaleY: -1}],
+    transform:
+      // It's important to invert the Y AND X axis to prevent a react native issue that can lead to ANRs on android 13
+      Platform.OS === 'android' ? [{scaleX: -1}, {scaleY: -1}] : [{scaleY: -1}],
   },
   horizontallyInverted: {
     transform: [{scaleX: -1}],
```

Comment:
Is the same workaround needed for horizontal lists?

Reply:
Horizontal lists don't need that workaround, there won't be any ANRs when inverted